### PR TITLE
Fix issue where AccessDenied ACE's were not being filtered out for se…

### PR DESCRIPTION
**Pull request by Coder666**
…rvice permission checks

### CODE REVIEW

File: SharpUp/Program.cs [hunk 1]
```diff
@@ -656,26 +656,29 @@ public static void GetModifiableServices()
 
                     foreach (System.Security.AccessControl.CommonAce ace in dacl)
                     {
-                        if (identity.Groups.Contains(ace.SecurityIdentifier))
+                        if (AceType.AccessAllowed == ace.AceType) //Filter allowed ace
```
Comment (cnotin) [inline SharpUp/Program.cs]: The other way around is clearer IMO:
`if (ace.AceType == AceType.AccessAllowed) //Filter allowed ace`

**Comment by cnotin** [comment]
I confirm this PR works well and allowed me to remove false positives due to AccessDenied ACEs, thanks @Coder666!
Just see comment above for a small suggestion

File: SharpUp/Program.cs (hunk 1 above)
Comment (Coder666) [reply SharpUp/Program.cs]: This is Yoda notation:  https://en.wikipedia.org/wiki/Yoda_conditions

There are, as with everything for and against arguments.  It depends how much you will be reading this line I guess.

Comment (cnotin) [reply SharpUp/Program.cs]: Interesting :)

**Pull request closed without merging**